api/config/v1/flags.go

```diff
@@ -101,11 +101,12 @@ func (f *deviceListStrategyFlag) UnmarshalJSON(b []byte) error {
 
 // GFDCommandLineFlags holds the list of command line flags specific to GFD.
 type GFDCommandLineFlags struct {
-	Oneshot         *bool     `json:"oneshot"         yaml:"oneshot"`
-	NoTimestamp     *bool     `json:"noTimestamp"     yaml:"noTimestamp"`
-	SleepInterval   *Duration `json:"sleepInterval"   yaml:"sleepInterval"`
-	OutputFile      *string   `json:"outputFile"      yaml:"outputFile"`
-	MachineTypeFile *string   `json:"machineTypeFile" yaml:"machineTypeFile"`
+	Oneshot               *bool     `json:"oneshot"         yaml:"oneshot"`
+	NoTimestamp           *bool     `json:"noTimestamp"     yaml:"noTimestamp"`
+	SleepInterval         *Duration `json:"sleepInterval"   yaml:"sleepInterval"`
+	OutputFile            *string   `json:"outputFile"      yaml:"outputFile"`
+	MachineTypeFile       *string   `json:"machineTypeFile" yaml:"machineTypeFile"`
+	VMPassthroughWorkload *bool     `json:"vmPassthroughWorkload" yaml:"vmPassthroughWorkload"`
```

Comment:
I would drop the `Workload` and maybe rename the flag completely to something with `vfio` in the name

Reply:
@klueska as I suggested below, adding a top-level mode flag to switch between this, `nvml`, and `cuda` explicitly may be useful. Do you have specific thoughts on this?